cve_bin_tool/checkers/firefox.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2022 Orange
```

Comment:
I'm not sure about the spdx header here.Usually we put Intel instead of "Orange" here. Terri might be able to share some insights into it.

Reply:
@terriko

Reply:
I'm guessing you probably either want to keep your copyright, in which case you can put your own name in there, or you want to assign to Intel.  (What you're doing right now is basically assigning copyright to Orange, which is also valid just probably not what you wanted to do.)  The important thing for me is mostly that the license line going with the copyright is compatible with the rest of the project.